meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+* RW: We allow specification to have a ??? under W3C license. I'd insist we keep ??? and have a disclaimer. The only thing we need to take is notification, which is a nuisance. It's no issue to transition even if you put code snippets as being public domain. I want to prevent some big corp coming, taking the work, making incompatible changes and claiming they can. What Tim wants is being able to take snippets and put in software with no hassle, but we still should be able to say something is not compatible with what the group consensus was. You have to include an adapted MIT license, I will help you do so. In transitioning to WG, I don't see any issue.
+* WT: Even if W3C licensing document says code snippets must be under software license, we still can change that license?
+* RW: Yes.
+* SC: How do we go from what we have in the charter right now to finalizing, merging charter after making clarification? Does anyone have any objections? Especially TBL. Does what RW suggests work for you? 
+* RW: To reiterate: the plan is to make a W3C software license with no notification but a disclaimer.
+* TBL: Happy to ??? but only for the code snippets that the notification is removed. I want what the WG produces to be usable by people. 
+* WT: RW, does what we currently have in charter accomplish what you suggest?
+* RW: What is in the charter is not in the document and the object of the rule is not clear. In the charter we can give the precise URL of a specification we can make assertions. Having MIT does not do the trick because of notification. I suggest we put W3C software license and in the annex say the code snippets are under W3C license without application of notification. I may want to create a W3C snippet license that does exactly what Tim wants but this will take a long time. There's no issue whatsoever with spec authors ??? I can give you a 3-liner for the charter so contributors commit to this. 
+* WT: I agree we should write whatever license we use in the documents. My question is given the text of the charter, which informs the line that should be in the documents, can we simply switch MIT to "W3C software without notice"? 
+* RW: Yes. 
+* RG: My question to RW is, is it already the case in copyright law that if you take a small portion of the work and you're doing it in a reasonable way you don't even need to worry about copyright because you're not using it as is? 
+* RW: If you never travel to Europe in your life you could operate this way. In the US you have fair use sometimes. 
```

Comment:
```suggestion
* RG: My question to RW is:
  Is it already the case in copyright law that, if you take a small portion of the work and you're doing it in a reasonable way, you don't even need to worry about copyright because you're not using it as is? 
* RW: If you never travel to Europe in your life, you could operate this way. In the US, you have fair use, sometimes. 
```